.changeset/clever-cats-jump.md

```diff
@@ -0,0 +1,10 @@
+---
+"fluid-framework": minor
+"@fluidframework/map": minor
+---
+
+DDS classes are no longer publicly exported
+
+ShareDirectory and SharedTree now only export their factories and the interface types.
```

Comment:
When did this change happen for SharedTree? Within the same version? We're about to branch RC2.

Reply:
It is a typo, already fixed